content/docs/sidebar.json

```
@@ -87,6 +87,19 @@
         "slug": "what-is-dvc",
         "source": "what-is-dvc.md"
       },
+      {
+        "label": "Concepts",
```

Will commit this for now:
```suggestion
        "label": "Basic Concepts",
```